src/components/Notifications/index.tsx

```diff
@@ -1,6 +1,5 @@
 import * as React from 'react';
 import ReactNotification, { store } from 'react-notifications-component';
-import 'react-notifications-component/dist/theme.css';
```

Comment:
Are the styles below everything that is in the theme?

Reply:
No they're just for the storybook as I was doing some tests on Etcher too. WRT porting them all: should I make a file inside the Notification component with the theme? Feels better than a `styles.ts` in the src folder or writing them inside index.ts of the component, which would make it quite uncomfortable to read

Reply:
No, I was just testing stuff in the storybook + Etcher, including them in a separate commit for semplicity